desktop/core/src/desktop/js/apps/storageBrowser/StorageBrowserPage/StorageBrowserTable/StorageBrowserPagination/StorageBrowserPagination.scss

```diff
@@ -0,0 +1,75 @@
+// Licensed to Cloudera, Inc. under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  Cloudera, Inc. licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+@use 'variables' as vars;
+@use 'mixins';
+
+.hue-storage-browser__pagination {
+  background-color: vars.$fluidx-gray-100;
+  height: 20px;
+  margin: 0;
+  padding: vars.$fluidx-spacing-s;
+  display: flex;
+  gap: vars.$fluidx-spacing-xxl;
+  justify-content: flex-end;
+  color: vars.$fluidx-gray-700;
+}
+
+.hue-pagination__page-size-control {
```

Comment:
"There are only two hard problems in Computer Science: cache invalidation and naming things — Phil Karlton"

In the tsx file the **hue-pagination** refers to the same block as **hue-storage-browser__pagination**. So _if_ we need a custom storage browser pagination I would name it:


```
// block
hue-storage-browser-pagination

// block__element
hue-storage-browser-pagination__page-size-control

```